packages/bitcore-node/src/providers/chain-state/internal/internal.ts

```diff
@@ -565,6 +565,24 @@ export class InternalStateProvider implements CSP.IChainStateService {
     }
   }
 
+  streamUnspentsByBlock(params: CSP.GetUnspentsFromBlockParams) {
+    const { req, res, args } = params;
+    const { limit, since, paging } = args;
+
+
+    if (typeof params.blockHeight !== 'number' || !params.chain || !params.network)
+      throw 'Missing required param';
+
+    let query = {
+      chain: params.chain,
+      network: params.network,
+      mintHeight: params.blockHeight
```

Comment:
It would be cool if we had a `includeSpent` arg, then you could do something like 
```
if(includeSpent) {
  query.$or = [{mintHeight: params.bockHeight}, {spentHeight: params.blockHeight}]
} else {
  query.mintHeight = params.blockHeight
}
```

Reply:
> It would be cool if we had a `includeSpent` arg, then you could do something with this effect
> 
> ```
> if(includeSpent) {
>   query.$or = [{mintHeight: params.bockHeight}, {spentHeight: params.blockHeight}]
> } else {
>   query.mintHeight = params.blockHeight
> }
> ```

Done.
Please check :)